pytensor/xtensor/type.py

```diff
@@ -505,6 +525,48 @@ def isel(
         return px.indexing.index(self, *indices)
 
     def set(self, value):
+        """Return a copy of the variable indexed by self with the indexed values set to y.
+
+        The original variable is not modified.
+
+        Raises
+        ------
+        ValueError
+            If self is not the result of an index operation
+
+        Examples
+        --------
+
+        .. testcode::
+
+            import pytensor.xtensor as ptx
+
+            x = ptx.as_xtensor([[0, 0], [0, 0]], dims=("a", "b"))
+            idx = ptx.as_xtensor([0, 1], dims=("a",))
+            out = x[:, idx].set(1)
+            out.eval()
+
+        .. test-output::
```

Comment:
The output version is also without hyphen: https://www.sphinx-doc.org/en/master/usage/extensions/doctest.html#directive-testoutput. It is also hidden from the rendered output due to unrecognized directive for now

Reply:
Is there no pre-commit we could use to fail on wrong directives?

Reply:
Feel free to check if number of warnings is crazy, that would be useful even if it doesn't address my problem